common/gpio.h

```diff
@@ -5,7 +5,7 @@
   #define GPIO_HUB_RST_N        30
   #define GPIO_UBLOX_RST_N      32
   #define GPIO_UBLOX_SAFEBOOT_N 33
-  #define GPIO_UBLOX_PWR_EN     34
+  #define GPIO_GNSS_PWR_EN      34
```

Comment:
can you keep the pin name from the schematic as a comment?

Reply:
sure, I added it as an inline comment here and in pins.py